examples/Authentication/google_ads_php.ini

```diff
@@ -23,6 +23,13 @@ developerToken = "INSERT_DEVELOPER_TOKEN_HERE"
 ; Required OAuth2 credentials. Uncomment and fill in the values for the
 ; appropriate flow based on your use case.
 
+; Set to true to use Application Default Credentials (ADC). When true, the library will attempt
+; to automatically find credentials based on the environment. This is often set up using
+; `gcloud auth application-default login`. ADC takes precedence over other OAuth2 settings
+; in this section if enabled.
+; See https://cloud.google.com/docs/authentication/application-default-credentials for details.
+; useApplicationDefaultCredentials = true
```

Comment:
Hmm, so in this case, the "default" would override the explicitly set `clientId`/`clientSecret`/`refreshToken` values below. I would suggest that the behavior should be the application default credentials kick in when the values below aren't found. If this can be done in a backwards-compatible way, we should just make that happen all the time (and not require a parameter set to true)